lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = { 'fish' => 5, 'chips' => 3, 'gravy' => 1, 'savoly' => 2 }
+  end
+
+  def read
+    @dishes
```

Comment:
I believe as it stands the read method will simply return the hash to the user. Could have used extrapolation to print out menu in a reader friendly way